Rewrite some text for clarity and flow.

wkumari opened the pull request:
Also fix whitespace, formatting, etc.

This is just initial (proposed) editorial fixes, and I'll send a PR with more substantive / useful ones soon. Obviously much of this is editorial opinion, feel free to spindle, fold, mutilate and / or reject....

wkumari committed:
Rewrite some text for clarity and flow.

Also fix whitespace, formatting, etc.

kjd commented on draft-davies-internal-tld.mkd:
```diff
@@ -55,63 +55,64 @@ use in private applications.
-IP address ranges that are set aside (e.g. {{RFC1918}}).
+as the network operator sees fit.
+
+The "internal" top-level domain is intened to fulfil a similar purpose to the IP address ranges that are set aside for private-use (e.g. {{RFC1918}}).
```
Typo. s/intened/intended/

wkumari replied:
Done! 
(I'm trying this using the GH web editor as I'm not at the correct machine to address).

wkumari committed:
Update draft-davies-internal-tld.mkd

Address typo change.

wkumari commented:
I had though that I had already addressed that, but perhaps it is in a later PR.
Anyway, I believe I addressed your comment -- but I'm trying do do this using the GitHub web UI so have no idea if it worked :-)

wkumari committed:
Fixed typo of 'intended'

wkumari committed:
Merge branch 'initial_editorial' of github.com:wkumari/fork_draft-davies-internal-tld into initial_editorial

wkumari commented:
Closing this PR as #13 has all of the changes consolidated into one.
